jobs/frontend/senior-frontend-developer.md

```diff
@@ -14,17 +14,38 @@ We're looking for a pro-active senior-level frontend developer, who not only sol
 ## We expect that you
 
 - Share our values ([en](https://datarockets.com/blog/company/datarockets-codex-core-values/), [ru](https://docs.google.com/document/d/1DojRoGnl-J0Ku0B9tjK_-HUMbKQDPoueoYr645dYA-4/edit)) – it's very important.
+- Write readable and maintainable code.
+- Care about user experience.
+- Have knowledge of best practices in development and in UI/UX design.
```

Comment:
I wanted to point that it's very important for frontend developer to always think about UI and UX.